github-actions/deploy-previews/pack-and-upload-artifact/action.yml

```diff
@@ -39,8 +39,7 @@ runs:
       id: copy
       shell: bash
       run: |
-        dir="$RUNNER_TEMP/pack-and-upload-tmp-dir"
-        mkdir -p $dir
+        dir="$RUNNER_TEMP/pack-and-upload-tmp-dir/"
```

Comment:
I am a bit confused about how this prevents the nesting?

Reply:
`cp A B` will create `B/A` if `B` already exists. If `B` would not exist, `A` will be copied directly as `B`

Reply:
Oh okay, didn't realize that. I think when I accidentally find myself in that situation I just erase and try again.